README.md

```diff
@@ -1,67 +1,58 @@
-[![Build Status](https://travis-ci.org/ornlneutronimaging/iMars3D.svg?branch=master)](https://travis-ci.org/ornlneutronimaging/iMars3D)
+<!-- Badges -->
```

Comment:
You can add the badge for [github-actions](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge).